docs/Routing.md

```diff
@@ -0,0 +1,86 @@
+# Routing
+
+[<< Back to index](./index.md)
+
+1. [Basic Routing Principles](#basic-routing-principles)
+2. [FirstSpirit Navigation Service](#firstspirit-navigation-service)
+3. [The triggerRouteChange Function](#the-triggerroutechange-function)
+
+## Basic Routing Principles
+
+In [Nuxt.js](https://nuxtjs.org) routing works via convention. You can link to any pages you may have created in the `pages` folder using the `<NuxtLink>` component. It has a prop called `to` that takes a route as parameter. If you have subfolders in the `pages` folder you can add them to the route as well.
+
+SFC-Example
+
+```vue
+<template>
+  <NuxtLink to="/">Home page</NuxtLink>
+  <NuxtLink to="/myPage">Links to MyPage.vue in the pages folder </NuxtLink>
+  <NuxtLink to="/aboutPages/myAboutPage">Links to MyAboutPage.vue in the pages/aboutPages folder.</NuxtLink>
+</template>
+```
+
+TSX-Example
+
+```typescript jsx
+
+render() {
+  declare const NuxtLink: Component
+  return (
+    <NuxtLink to="/">Home Page</NuxtLink>
+  )
+}
+```
+
+## FirstSpirit Navigation Service
```

Comment:
This topic should be on top of the page, since it is our way to build the site structure. 
Maybe you could add a screenshot of the navigation structure and explain where the matching content is coming from.